streaming/streaming.go

```diff
@@ -345,8 +344,9 @@ func LaunchDataflowJob(ctx context.Context, targetProfile profiles.TargetProfile
 			"deadLetterQueueDirectory": inputFilePattern + "dlq",
 		},
 		Environment: &dataflowpb.FlexTemplateRuntimeEnvironment{
-			NumWorkers: numWorkers,
-			MaxWorkers: maxWorkers,
+			MaxWorkers:            maxWorkers,
+			AutoscalingAlgorithm:  2, // 2 corresponds to AUTOSCALING_ALGORITHM_BASIC
```

Comment:
What will be the initial number of workers in this case ?

Reply:
It starts with 1, then every 10 minutes, the autoscaler re-scales.